### PR TITLE
Ask to re-install with `--unsafe-perm` on `EACCES`

### PR DESCRIPTION
**Pull request by igorklopov**

### CODE REVIEW

File: download/install.js [hunk 1]
```diff
@@ -13,4 +13,6 @@ if (fs.existsSync(src)) {
   process.exit(0)
 }
 
+// Hack to force chalk display colors
+process.argv.push('--color');
```
Comment (Qix-) [inline download/install.js]: Why should colors be forced here?

Also, this is preferable:

```javascript
process.env.FORCE_COLOR = '1';
```

File: download/src/log.js [hunk 2]
```diff
@@ -20,6 +20,10 @@ export function info(text) {
   console.log(`> ${text}`)
 }
 
+export function warn(text) {
+  console.log(`${chalk.red('> Warning!')} ${text}`)
```
Comment (Qix-) [inline download/src/log.js]:
```javascript
console.log(chalk.red('> Warning!'), text);
```

Comment (igorklopov) [reply download/src/log.js]: Just copied from here https://github.com/zeit/now-cli/blob/7.0.1/lib/utils/output/error.js#L8 following code conventions.

File: download/install.js (hunk 1 above)
Comment (igorklopov) [reply download/install.js]: Colors are needed for red warning https://github.com/zeit/now-cli/blob/eacces-rerun-with-unsafe-perm/download/src/log.js#L24. You are right in general but if you try to run your process.env code, it does not work.

**Comment by leo** [comment]
Thanks a lot @igorklopov! 👌

**Pull request merged**